src/pages/MapViewer/components/Toolbox/components/item-assets/AssetsPanel.tsx

```diff
@@ -0,0 +1,49 @@
+import { ClipboardButton } from '@/components/clipboard/ClipboardButton';
+import { useToolbox } from '@/hooks/useToolbox';
+
+import './AssetsPanel.scss';
+
+export const AssetsPanel = () => {
+  const {
+    state: { selectedCollectionItem },
+    actions: { setActivePage },
+  } = useToolbox();
+
+  // Reduce the object of objects down to an array of key/value objects.
+  const assets = Object.keys(selectedCollectionItem.assets).reduce((acc, key) => {
+    acc = [...acc, { key, value: selectedCollectionItem.assets[key].href }];
+
+    return acc;
+  }, []);
+
+  return (
+    <div>
+      <div className="toolbox__header">
+        <button
+          className="button-link"
+          onClick={() => {
+            setActivePage('items');
+          }}
+        >
+          <span>&lt; Return to Items</span>
+        </button>
+      </div>
+
+      <ul>
+        {assets && assets.length > 0 ? (
+          assets.map((asset) => (
```

Comment:
Typed as mentioned above with the `StacAsset`

Reply:
See comment above on this. At best, this would be a new type as I stated, not a `StacAsset`.